share/smack/top.py

```diff
@@ -339,6 +339,8 @@ def llvm_to_bpl(args):
 
   cmd = ['llvm2bpl', args.linked_bc_file, '-bpl', args.bpl_file]
   cmd += ['-warn-type', args.warn]
+  cmd += ['-sea-dsa=ci']
```

Comment:
Why do we turn this on even when user selects `no-memory-splitting`?

Reply:
Regions.cpp is not the only place where dsa is needed. CodifyStaticInits.cpp also relies on DSA. Furthermore, we have code that bypasses DSA when `no-memory-splitting` is enabled.

Reply:
I see. Ok, makes sense.